src/Compilers/VisualBasic/Test/Symbol/SymbolsTests/Source/EventTests.vb

```diff
@@ -2209,5 +2209,26 @@ End Enum
             Assert.Equal(0, symbolInfo.CandidateSymbols.Length)
         End Sub
 
+        <WorkItem(9400, "https://github.com/dotnet/roslyn/issues/9400")>
+        <Fact()>
+        Public Sub HandlesNoMyBase()
+            Dim compilation = CompilationUtils.CreateCompilationWithMscorlib(
+<compilation>
+    <file name="a.vb"><![CDATA[
+Interface I
```

Comment:
I think the repro in the issue uses a class. Is it a mistake in the issue? Let's correct it, otherwise consider adding a test with a class.